questions/includes/openSUSE/coding/web.yml

```diff
@@ -0,0 +1,9 @@
+tree:
+  children:
+    - title: Website
```

Comment:
Which website?

Reply:
Was thinking of more than just the landing page. We hadn't completely gone done that route for how to go about doing that. I think it can be resolved through the wiki as you suggest.